Week-3/exercise3_1.js

```diff
@@ -0,0 +1,12 @@
+const cache = {};
+const add = (...args) => args.reduce((a, b) => a + b, 0);
```

Comment:
what if there are no args?

Start thinking of corner cases and edge cases , which is critical for good DSA understanding